mesonbuild/modules/gnome.py

```diff
@@ -151,8 +152,8 @@ def compile_resources(self, state, args, kwargs):
         else:
             raise MesonException('Invalid file argument: {!r}'.format(ifile))
 
-        depend_files, depends, subdirs = self._get_gresource_dependencies(
-            state, ifile, source_dirs, dependencies)
+        depend_files, depends, subdirs = self._get_gresource_dependencies(glib_compile_resources,
+            state, ifile, source_dirs, dependencies, )
```

Comment:
__[Flake8]__ [E128] continuation line under-indented for visual indent ([link](https://sider.review/gh/repos/19784232/pulls/5772#issue-4412201))

<sub>posted by Sider</sub>